### PR TITLE
AC Stub Test - LRAC-8352 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LRAC-8352

### CODE REVIEW

**Comment by AfonsoHSantos** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesAcquisitions.testcase [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+		}
+
+		ACUtils.tearDownAC();
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8344 | Test Summary: Site overview acquisition card shows which channels individuals are coming from"
+	@ignore = "true"
+	@priority = "4"
+	test AssertShowsChannelsIndividualsComingFrom {
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesAcquisitions.testcase:36]: hi @AfonsoHSantos, it is a priority 5 on the test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesAcquisitions.testcase [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8346 | Test Summary: Assert Sites Acquisition card Channels highest session"
+	@ignore = "true"
+	@priority = "4"
+	test AssertSitesAcquisitionCardChannelsHighestSession {
+
+		// TODO LRAC-8346 AssertSitesAcquisitionCardChannelsHighestSession
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesAcquisitions.testcase:47]: hi @AfonsoHSantos , it is a priority 3 on the test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesAcquisitions.testcase [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8347 | Test Summary: Assert Sites Acquisition card Referrers highest session"
+	@ignore = "true"
+	@priority = "4"
+	test AssertSitesAcquisitionCardReferrersHighestSession {
+
+		// TODO LRAC-8347 AssertSitesAcquisitionCardReferrersHighestSession
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesAcquisitions.testcase:55]: hi @AfonsoHSantos , it is a priority 3 on the test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesAcquisitions.testcase [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8348 | Test Summary: Assert Sites Acquisition card Source|Medium highest session"
+	@ignore = "true"
+	@priority = "4"
+	test AssertSitesAcquisitionCardSourceAndMediumHighestSession {
+
+		// TODO LRAC-8348 AssertSitesAcquisitionCardSourceAndMediumHighestSession
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesAcquisitions.testcase:65]: hi @AfonsoHSantos , it is a priority 3 on the test map.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesAcquisitions.testcase [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8345 | Test Summary: Assert Sites Acquisition change time filter"
+	@ignore = "true"
+	@priority = "4"
+	test AssertSitesAcquisitionChangeTimeFilter {
+
+		// TODO LRAC-8345 AssertSitesAcquisitionChangeTimeFilter
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesAcquisitions.testcase:74]: hi @AfonsoHSantos , it is a priority 3 on the test map.

**Review by alexcandidos — CHANGES_REQUESTED** [review]
hi @AfonsoHSantos , please take a look at my comments. Thanks!

**Comment by alexcandidos** [comment]
Also, the PR and commits are using the wrong ticket id, it should be LRAC-8338 (the ticket we created to commit the stubs) instead of LRAC-8338 (related to the task).

**Comment by AfonsoHSantos** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @AfonsoHSantos , Thanks for working on it!

The code looks good to me!

**Comment by alexcandidos** [comment]
ci:forward

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**